vulnerabilities/api.py

```diff
@@ -177,7 +176,7 @@ def bulk_search(self, request):
 
 
 class VulnerabilityFilterSet(filters.FilterSet):
-    vulnerability_id = filters.CharFilter(field_name="cve_id")
+    vulnerability_id = filters.CharFilter(field_name="vulnerability_id")
```

Comment:
`field_name` seems unneeded since duplicated from the attribute.

Reply:
Good catch :) fixing this